packages/framework/src/Model/Order/Order.php

```diff
@@ -1028,13 +1028,34 @@ public function edit(
                 $newOrderItemData->unitName,
                 $newOrderItemData->catnum
             );
+            if (!$newOrderItemData->usePriceCalculation) {
+                $newOrderItem->setTotalPrice(new Price($newOrderItemData->totalPriceWithoutVat, $newOrderItemData->totalPriceWithVat));
+            }
         }
 
         $this->calculateTotalPrice($orderPriceCalculation);
 
         return new OrderEditResult($statusChanged);
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData $orderItemData
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemPriceCalculation $orderItemPriceCalculation
+     */
+    protected function editOrderItemDataPrices(OrderItemData $orderItemData, OrderItemPriceCalculation $orderItemPriceCalculation): void
```

Comment:
This method invoked me something completely different... I thought it is about editing OrderItemPrices, but it's about data.... Please rename it to something like `calculateOrderItemDataPrices`

Reply:
Renamed in 2fcd799d0